### PR TITLE
Add exportSnapshot support

### PR DESCRIPTION
**Pull request by jeromer**

### CODE REVIEW

**Comment by falzm** [comment]
PR fixed to actually return the pre-signed URL to the caller: 10f3c6a.

File: Makefile [hunk 1]
```diff
@@ -32,3 +32,9 @@ publish-doc: apidoc html
 
 serve:
 	@python3 -m http.server 8000 --bind 127.0.0.1 --directory $(BUILDDIR)/html
+
+installrequirements:
+	@pip install wheel
+	@pip install -r requirements.dev.txt
+	@pip install -r requirements.txt
+	@pip install -e .
```
Comment (brutasse) [inline Makefile:40]:
```suggestion
	@pip install -e .
```

(installing `.` should pull the base requirements)

File: exoscale/api/compute.py [hunk 2]
```diff
@@ -1035,6 +1035,21 @@ def delete(self):
...
+        Returns:
+            str: the URL of the exported snapshot file
+        """
+
+        try:
+            res = self.compute.cs.exportSnapshot(id=self.id)
+        except CloudStackApiException as e:
+            raise APIException(e.error["errortext"], e.error)
+
+        return res["presignedurl"]
```
Comment (brutasse) [inline exoscale/api/compute.py:1051]:
```suggestion
        return res
```

It seems there won't always just be a pre-signed URL in the response, but possibly some more metadata.

Comment (falzm) [reply exoscale/api/compute.py]: Changed in bf307f2

**Pull request merged**